src/slotted_page.rs

```diff
@@ -237,6 +268,35 @@ impl SlottedPageMmap {
         data_start_offset.saturating_sub(last_slot_offset)
     }
 
+    pub fn page_size(&self) -> usize {
+        self.header.page_size()
+    }
+
+    /// Sums the amount of unused space in between the data.
+    pub fn fragmented_space(&self) -> usize {
```

Comment:
Can you maybe write a simple test for this one?
Write a few values and then delete one and check that fragmentation increases.

Reply:
Ah yes, I was forgetting about this one

Reply:
Done (https://github.com/qdrant/mmap-payload-storage/pull/3/commits/ced191838fd6dcca19a239260013b6df66f91be0)